src/fuzz/protobufs/utils/spanner_emulator_ddl_statement_proto_to_string.h

```diff
@@ -0,0 +1,46 @@
+//
```

Comment:
Ahh, I missed something here.

We'll want to open this file after the license header with an 
#ifndef SRC_FUZZ_PROTOBUF_UTILS_SPANNER_EMULATOR_DDL..._H
#define SRC_FUZZ..._H

and then end with

#endif // SRC_FUZZ_..._H